api/v1alpha1/virtualmachinepublishrequest_types.go

```diff
@@ -215,6 +218,11 @@ type VirtualMachinePublishRequestStatus struct {
 	// +optional
 	StartTime metav1.Time `json:"startTime,omitempty"`
 
+	// Attempt represents the number of times the request has been attempted.
+	//
+	// +optional
+	Attempt int `json:"attempt,omitempty"`
```

Comment:
Please change this to 

```golang
Attempts int `json:"attempts,omitempty"`
```

Reply:
Changed. 👍 Also based on @aruneshpa 's suggestion in today's meeting, add a `lastAttemptTime` field to expose more details to users.